README.md

```diff
@@ -1,9 +1,92 @@
 # Reactive GraphQL
+> GraphQL reactive executor
```

Comment:
I have the feeling that "reactive executor" could be a confusing term which might prevent people from using it.

```suggestion
> GraphQL implementation providing a reactive API based on RxJS
```

Reply:
What do you think about this change?

Reply:
Yes works for me!

The fact that is based on RxJS is a technique of implementation, and therefore a detail IMO 🤷‍♂️

I don't know how you want to handle this: IMO, many people looking for a GraphQL reactive implementation are directing their search to a implementations of _live queries_. Even if reactive-graphql is not strictly an implementation of it, I think making a link between _live queries_ and reactive-graphql would help.